core/integration/features/verified/Negative-Position-Decimal-Places.feature

```diff
@@ -0,0 +1,172 @@
+Feature: Allow markets to be specified with a smaller number of decimal places than the underlying settlement asset
```

Comment:
Not sure I understand how this description matches the scenarios? How do negative position decimal places relate to the underlying asset?